scripts/rebuilder.py

```diff
@@ -381,14 +356,14 @@ def main():
         project = copr_client.project_proxy.get(project_owner, project_name)
         for repo in project['additional_repos']:
             match = re.match(r"copr://@fedora-llvm-team/llvm-snapshots-big-merge-([0-9]+)$", repo)
-            if not match:
-                continue
-            print(datetime.datetime.fromisoformat(match.group(1)).isoformat())
+            if match:
+                print(datetime.datetime.fromisoformat(match.group(1)).isoformat())
+                return
     elif args.command == "rebuild-in-progress":
         for pkg in copr_client.monitor_proxy.monitor(project_owner, project_name)["packages"]:
             for c in pkg["chroots"]:
-                chroot = pkg["chroots"][c]
-                if chroot["state"] not in ["succeeded", "forked", "skipped", "failed", "canceled"]:
+                build = CoprBuild(pkg["chroots"][c])
```

Comment:
The Copr API looks very weird, doesn't it? I mean you're iterating over chroots and then instantiate a `CoprBuild` variable from a chroot? Strange.